docs-v2/sdks/node.mdx

```diff
@@ -338,23 +336,32 @@ console.log(records);
 By default this returns an array of objects in the data model that you queried with some metadata about each record.
 
 ```json
-[
-    {
-        id: 123,
-        ..., // Fields as specified in the model you queried
-        _nango_metadata: {
-            deleted_at: null,
-            last_action: 'ADDED',
-            first_seen_at: '2023-09-18T15:20:35.941305+00:00',
-            last_modified_at: '2023-09-18T15:20:35.941305+00:00'
-        }
-    },
-    ...
-]
+{
+    records:
+        [
+            {
+                id: 123,
+                ..., // Fields as specified in the model you queried
+                _nango_metadata: {
+                    deleted_at: null,
+                    last_action: 'ADDED',
+                    first_seen_at: '2023-09-18T15:20:35.941305+00:00',
+                    last_modified_at: '2023-09-18T15:20:35.941305+00:00'
+                }
+            },
+            ...
+        ],
+    next_cursor: "Y3JlYXRlZF9hdF4yMDIzLTExLTE3VDExOjQ3OjE0LjQ0NyswMjowMHxpZF4xYTE2MTYwMS0yMzk5LTQ4MzYtYWFiMi1mNjk1ZWI2YTZhYzI"
```

Comment:
Is the cursor still going to be this long??

Reply:
Yeah, the content is fairly long. Timestamp + uuid

for example:
```
2023-11-17T11:47:14.447+02:00||012d21e7-727a-446c-8a06-96847fa31f86
```